ethdb/memory_database.go

```diff
@@ -88,12 +97,15 @@ func (db *MemDatabase) Delete(key []byte) error {
 	return nil
 }
 
+// Close performs no operation but imitates invocation of levelDB.Close().
```

Comment:
`// Close implements the closer interface. For the memory database, this is a noop.`